lib/cli.js

```diff
@@ -23,19 +23,19 @@ module.exports = function broccoliCLI(args) {
     .description('start a broccoli server')
     .option('--port <port>', 'the port to bind to [4200]', 4200)
     .option('--host <host>', 'the host to bind to [localhost]', 'localhost')
-    .option('--brocfile-path <path>', 'the path to brocfile')
+    .option('--brocfile <path>', 'the path to brocfile')
```

Comment:
We decided to change this to `--brocfile-path`, can you bring that back?

Reply:
done. can you please update https://github.com/broccolijs/broccoli/issues/331 to reflect your decisions. 
I plan to implement other params now, so would like to be sync :-)

Reply:
No problem!  I updated it (we only discussed `--brocfile-path` though)...